src/ocsp.c

```diff
@@ -383,6 +383,24 @@ static int CheckResponse(WOLFSSL_OCSP* ocsp, byte* response, int responseSz,
     return ret;
 }
 
+WOLFSSL_API int wolfSSL_CertManagerCheckOCSP_Staple(WOLFSSL_CERT_MANAGER *cm,
+                                                    byte *response, int responseSz, buffer *responseBuffer,
```

Comment:
Whitespace change request: could you shift these lines to the left so they don't go past column 80? Changing these lines to an indent of 8 spaces would be good.

Reply:
Fixed